Update with new PowerShell cmds storage-service-encryption-customer-managed-keys.md

AngshumanNayakMSFT opened the pull request:
Step 4 : Added PowerShell commands to associate an existing storage account with a KeyVault Key to be used for SSE with CMK. 

storageAccount = Get-AzureRmStorageAccount -ResourceGroupName "RGSSECMK" -AccountName "storagetousecmk"
$keyVault = Get-AzureRmKeyVault -VaultName "cmkwithsettings"
$key = Get-AzureKeyVaultKey -VaultName $keyVault.VaultName -Name "keytoencrypt"
Set-AzureRmKeyVaultAccessPolicy -VaultName $keyVault.VaultName -ObjectId $storageAccount.Identity.PrincipalId -PermissionsToKeys wrapkey,unwrapkey,get
Set-AzureRmStorageAccount -ResourceGroupName $storageAccount.ResourceGroupName -AccountName $storageAccount.StorageAccountName -EnableEncryptionService "Blob" -KeyvaultEncryption -KeyName $key.Name -KeyVersion $key.Version -KeyVaultUri $keyVault.VaultUri

azurecla commented:
Hi __@mattrowmsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mattrow). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

mattrowmsft commented:
#sign-off

Jim-Parker commented:
@mattrowmsft Matt, can you submit the pull request through the private repo? Thanks.

mattrowmsft commented:
Done

AngshumanNayakMSFT committed:
Update storage-service-encryption-customer-managed-keys.md

Step 4 : Added PowerShell commands to associate an existing storage account with a KeyVault Key to be used for SSE with CMK. 

storageAccount = Get-AzureRmStorageAccount -ResourceGroupName "RGSSECMK" -AccountName "storagetousecmk"
$keyVault = Get-AzureRmKeyVault -VaultName "cmkwithsettings"
$key = Get-AzureKeyVaultKey -VaultName $keyVault.VaultName -Name "keytoencrypt"
Set-AzureRmKeyVaultAccessPolicy -VaultName $keyVault.VaultName -ObjectId $storageAccount.Identity.PrincipalId -PermissionsToKeys wrapkey,unwrapkey,get
Set-AzureRmStorageAccount -ResourceGroupName $storageAccount.ResourceGroupName -AccountName $storageAccount.StorageAccountName -EnableEncryptionService "Blob" -KeyvaultEncryption -KeyName $key.Name -KeyVersion $key.Version -KeyVaultUri $keyVault.VaultUri

AngshumanNayakMSFT committed:
Update storage-service-encryption-customer-managed-keys.md

Updated the names as per Tamara's suggestions. 

"Would you mind making the names of the resource group, storage account, etc., non-specific? For example, replace
"RGSSECMK"
with
and so on.
"
